src/components/NostoProduct.tsx

```diff
@@ -29,6 +29,20 @@ export default function NostoProduct(props: {
   product: string
   tagging?: Product
   placements?: string[]
+}) {
+  useNostoProduct(props)
+  return null
+}
+
+/**
+ * You can personalise your product pages by using the useNostoProduct hook.
+ * 
+ * @group Hooks
+ */
+export function useNostoProduct(props: {
+  product: string
+  tagging?: Product
+  placements?: string[]
 }) {
```

Comment:
extract {
  product: string
  tagging?: Product
  placements?: string[]
} to type and then we can destructure it in method param?